src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personName = `My name is ${name}`;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const marriage = isMarried
+    ? `${partnerName} is my ${partner}`
+    : 'I am not married';
+
+  const ageIsIndicated = age
```

Comment:
Better delete it here and move it to return,
`  {age &&
        <p className="Person__age">{`I am ${age}`}</p>
     }`

Reply:
I think that I made it more clear and readable than you suggest.

Reply:
@DimaKobzar7  you can avoid of creation of a new variable and use this condition inside layout